### ISSUE TITLE
tag <li></li> makes a new line when turn into pdfmake, which doesn't happen when display html on browser

### ISSUE THREAD

**Issue by Leo16012000** [Aymkdn/html-to-pdfmake#147]
```
<ol>
<li>
<span>Line 1</span>
</li>
<li>
<p>Line 2</p>
</li>
<li>
<p><span>Line 3</span></p>
</li>
</ol>
```
What I expect:
![MicrosoftTeams-image](https://user-images.githubusercontent.com/67458956/180795968-e471d6ea-983c-416d-a5c6-86a21a8bf6be.png)

What I got: 
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/67458956/180796118-ce5475aa-2de1-460c-9627-f5c977cd9df2.png)

Is there anyway to solve my problem?

**Comment by Aymkdn** [comment]
Please, provide *at least* the HTML code you're using, and make some effort to write the issue's description…

**Comment by Aymkdn** [comment]
I've just published v2.4.5 that resolves this issue, but you need to apply the [option `removeExtraBlanks:true`](https://github.com/Aymkdn/html-to-pdfmake#removeextrablanks) when you call the function:
![image](https://user-images.githubusercontent.com/946315/180832812-bf94a56b-4845-4267-9784-152c058da880.png)

The other solution (without using `removeExtraBlanks:true`) is to remove the line breaks in your HTML code.
The code will then look like the below:
```html
<ol>
   <li><span>Line 1</span></li>
   <li><p>Line 2</p></li>
   <li><p><span>Line 3</span></p></li>
</ol>
```
